.github/workflows/main.yml

```diff
@@ -11,31 +11,28 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Build
         run: docker build -t aspera-client .
       - name: Test
-        run: docker run --rm aspera-client idr0001 md5sum.txt /tmp/idr0001/
+        run: docker run --rm aspera-client idr0001-graml-sysgro/md5sum.txt /tmp/idr0001/
   upload:
     needs: build
     runs-on: ubuntu-latest
     env:
       name: imagedata/download
     steps:
-      - name: Get prefix
-        id: getprefix
-        run: |
-          if [ ! -z ${{ env.name }} ]; then
-            echo "::set-output name=prefix::${{ env.name }}:"
-          else
-            echo "::set-output name=prefix::${{ github.repository }}:"
-          fi
       - name: Get other tags
         id: gettags
-        uses: jupyterhub/action-major-minor-tag-calculator@v1.1.0
+        uses: jupyterhub/action-major-minor-tag-calculator@v3
         with:
           githubToken: ${{ secrets.GITHUB_TOKEN }}
-          prefix: "${{ steps.getprefix.outputs.prefix }}"
+          prefix: "${{ env.name }}"
```

Comment:
Looks like I missed there was a breaking change and a `:` need to be appended as in https://github.com/ome/omero-server-docker/commit/6f317379d5a5d191200a491281862a334bd349c3.

Unfortunately, this has created quite a few unwanted images at https://hub.docker.com/u/imagedata. @jburel @joshmoore do you have the credentials to delete those? I can open a PR for a follow-up 0.4.1 release (unless you'd prefer me to delete the tag).